src/NServiceBus.Gateway.RavenDB.AcceptanceTests/GatewayTestSuiteConstraints.cs

```diff
@@ -0,0 +1,84 @@
+﻿using NServiceBus.AcceptanceTesting.Support;
+using NServiceBus.Configuration.AdvancedExtensibility;
+using NServiceBus.Gateway.RavenDB;
+using Raven.Client.Documents;
+using Raven.Client.ServerWide;
+using Raven.Client.ServerWide.Operations;
+using System;
+using System.Threading.Tasks;
+
+namespace NServiceBus.Gateway.AcceptanceTests
+{
+    public partial class GatewayTestSuiteConstraints
+    {
+        public Task ConfigureDeduplicationStorage(string endpointName, EndpointConfiguration configuration, RunSettings settings)
+        {
+            var ravenGatewayDeduplicationConfiguration = new RavenGatewayDeduplicationConfiguration((builder, _)=> 
+            {
+                databaseName = Guid.NewGuid().ToString();
+                var documentStore = GetInitializedDocumentStore(databaseName);
+
+                documentStore.Maintenance.Server.Send(new CreateDatabaseOperation(new DatabaseRecord(databaseName)));
+
+                return documentStore;
+            });
+
+            var gatewaySettings = configuration.Gateway(ravenGatewayDeduplicationConfiguration);
+            configuration.GetSettings().Set(gatewaySettings);
+
+            return Task.FromResult(false);
```

Comment:
Doesn't Raven 4 force you into net461 minimum?
```suggestion
            return Task.CompletedTask;
```

Reply:
Addressed